tools/perspective-scripts/publish.mjs

```diff
@@ -207,6 +216,23 @@ if (proceed.toLowerCase() === "y") {
         recursive: true,
     });
 
+    const release = await octokit.request(
```

Comment:
`COMMIT` flags is a safety bool to prevent this script from having side effects unless intended. When run without `COMMIT`, I expect this script to run but not actually _publish_ in the end, so I was quite surprised when I ran this for this first time and it published a GitHub release.

Since this script is now being called from CI, the release-to-GitHub functionality in this script either needs to go behind the same `COMMIT` flag (see bottom of this script), or a separate flag just for CI.